src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -120,6 +121,7 @@ private static void setupWindows(Application application) {
         CommandManagerWindowCommandChangeObserver windowObserver = new CommandManagerWindowCommandChangeObserver(
                 commandManager);
         CommandsFeature.getDriverCommandManager().getChangePublisher().addSubscriber(windowObserver);
+        application.addTest("Transform current command", new TransformCurrentCommandOptionListener());
```

Comment:
You could add details concerning transformation (e.g. 1.5x 30deg x+50 y+25).
It would be nice to have a set of useful transformation and their reverse versions.